src/main/java/com/uploadcare/api/Client.java

```diff
@@ -252,26 +567,135 @@ public void close() {
     }
 
     /**
-     * @param fileId  Resource UUID
+     * @deprecated Use {@link #copyFileLocalStorage(String, Boolean, Boolean)}
+     * or {@link #copyFileRemoteStorage(String, String, Boolean, String)} instead.
+     *
+     * @param source  File Resource UUID or A CDN URL.
      * @param storage Target storage name
+     *
+     * @return CopyFile resource with result of Copy operation
+     */
+    @Deprecated
+    public CopyFile copyFile(String source, String storage) {
+        if (storage != null && !storage.isEmpty()) {
+            return copyFileRemoteStorage(source, storage, true, null);
+        } else {
+            return copyFileLocalStorage(source, true, true);
+        }
+    }
+
+    /**
+     * Copy file to local storage. Copy original files or their modified versions to default storage. Source files MAY
+     * either be stored or just uploaded and MUST NOT be deleted.
+     *
+     * @param source     File Resource UUID or A CDN URL.
+     * @param store      The parameter only applies to the Uploadcare storage and MUST be either true or false.
+     * @param makePublic Applicable to custom storage only. MUST be either true or false. true to make copied files
+     *                   available via public links, false to reverse the behavior.
+     *
      * @return An object containing the results of the copy request
      */
-    public CopyFileData copyFile(String fileId, String storage) {
+    public CopyFile copyFileLocalStorage(String source, Boolean store, Boolean makePublic) {
+        CopyOptionsData copyOptionsData = new CopyOptionsData();
+        copyOptionsData.source = source;
+        copyOptionsData.store = store;
+        copyOptionsData.makePublic = makePublic;
```

Comment:
this does not make sense
```suggestion
    public CopyFile copyFileLocalStorage(String source, Boolean store) {
        CopyOptionsData copyOptionsData = new CopyOptionsData();
        copyOptionsData.source = source;
        copyOptionsData.store = store;
```

Reply:
Fixed in 33471478